articles/hints_tips/UsefulTips/UsefulTipsApiArticle.md

```diff
@@ -0,0 +1,253 @@
+# How to implement the Studio Useful Tips service
+
+The `RwsAppStore.UsefulTips.Service` is a service provider for updating the Useful Tips that are displayed in Trados Studio 2019+.
+
+***
+
+## Add the Studio Useful Tips service to your project
+You can add the nuget package to your project via the package manager user interface or console.
+
+### Package Manager UI
+* In Solution Explorer, right-click References and choose Manage NuGet Packages.
+* Choose nuget.org as the Package source.
+* Search for `RwsAppStore.UsefulTips.Service` from the Browse tab.
+* Choose the package from the list and select Install.
+* Accept any license prompts.
+<img style="display:block; " src="images/InstallFromNuget.png" />
+
+### Package Manager Console 
+* Alternatively, select the Tools > NuGet Package Manager > Package Manager Console menu command.
+* From the Package Manager Console, enter the command:
+`Install-Package RwsAppStore.UsefulTips.Service -Version 2.1.9.5`
+
+***
+  
+## Remarks
+
+When adding tips, the Useful Tips service first checks if they already exist in the Useful Tips collection from Studio and then attempts to add only those that are identified as new.  
+Depending on whether or not Studio was launched as administrator, the user may receive a message from the service indicating a requirement to elevate the user rights prior to updating the Useful Tips collection in Studio with the new tips from the plugin.  
+> [!Note]
+> It is required to elevate the user rights to administrator, as the locale tip files that manage the Useful Tips collection in Studio reside in the Trados Studio 2019+ installation directory.  
```

Comment:
Please use <var:ProductName> instead of direct mention to Trados Studio 2019+ so we can dynamically change product name in the future if the case arises.